### PR TITLE
Update instructions and add configs for remaining CPU apps

### PR DESCRIPTION
**Pull request by milroy**
Update parallelcluster instructions and add configs for mixbench, mt-gemm, nekrs, osu, quicksilver, and stream.

### CODE REVIEW

File: experiments/aws/parallel-cluster/cpu/experiment/README.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ ssh -i ~/.ssh/milroy1-ldrd-east2-ed2.pem ubuntu@ec2-18-117-224-238.us-east-2.com
 Get the topology:
 
 ```bash
-aws ec2 describe-instance-topology --region us-east-2 --filters Name=instance-type,Values=hpc6a.48xlarge --filters Name=tag-key,Values=cluster-tag-value > topology-2.json
-aws ec2 describe-instances --filters "Name=instance-type,Values=hpc6a.48xlarge" --region us-east-2 > instances-2.json
+aws ec2 describe-instance-topology --region us-east-2 --filters Name=instance-type,Values=hpc6a.48xlarge --filters Name=tag-key,Values=cluster-tag-value > topology.json
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/README.md:75]: I've been keeping the topology "tagged" with the max size. Assuming this is for CPU (256 max):
```suggestion
aws ec2 describe-instance-topology --region us-east-2 --filters Name=instance-type,Values=hpc6a.48xlarge --filters Name=tag-key,Values=cluster-tag-value > topology-256.json
```

File: experiments/aws/parallel-cluster/cpu/experiment/README.md [hunk 2]
```diff
@@ -146,7 +143,7 @@ oras push ghcr.io/converged-computing/metrics-operator-experiments/performance:a
 
 #### AMG2023
 
-Since this container requires sourcing spack, we need to write a bash script to run on the host.
+All the following examples are for 32 nodes. Mutatis mutandis for other sizes.
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/README.md:146]: :crossed_swords: https://en.wikipedia.org/wiki/X-Mansion

File: experiments/aws/parallel-cluster/cpu/experiment/README.md [hunk 3]
```diff
@@ -198,85 +194,63 @@ Don't forget to save the MiniFE yaml output files that generate in the PWD.
...
 
-This wasn't on the path.
-
 ```bash
-# 49.78 seconds
-time singularity pull docker://ghcr.io/converged-computing/metric-mixbench:libfabric-cpu
-
-# 24 seconds
-time mpirun -np 2 --hostfile ./hostfile.txt /shared/bin/singularity exec /shared/containers/metric-mixbench_libfabric-cpu.sif mixbench-cpu
+cd configs/mixbench/
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/README.md:198]: Let's make sure to chat with Abhik, Tapasya, about what to do for iterations based on runtimes at different scales. I'm not sure the best way to handle that with a submit vs. interactive run.

File: experiments/aws/parallel-cluster/cpu/experiment/configs/nekrs/slurm-nekrs-128n.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+#SBATCH --nodes=128
+#SBATCH --time=0:10:00
+#SBATCH --exclusive
+
+echo "Start time:" $( date +%s )
+cd /shared/nekrs/turbPipe
+/usr/bin/time -p mpirun -N 128 --map-by ppr:96:node \
+	/shared/bin/singularity exec --pwd /shared/nekrs/turbPipe/ \
+	/shared/containers/metric-nek5000_libfabric-cpu.sif \
+	nekrs --setup /shared/nekrs/turbPipe/turbPipe.par
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/configs/nekrs/slurm-nekrs-128n.sh:13]: :hankey: :smoking:

File: experiments/aws/parallel-cluster/cpu/experiment/README.md (hunk 1 above)
Comment (milroy) [reply experiments/aws/parallel-cluster/cpu/experiment/README.md:75]: Done.

**Comment by milroy** [comment]
Updated the PR per your recommendations!

**Review by vsoch — APPROVED** [review]

**Pull request merged**